.github/workflows/test.yml

```diff
@@ -53,26 +53,24 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        name: [
-          "windows-py38",
-          "windows-py38-pluggy",
-          "windows-py39",
+        names: [
+          "windows-py39-unittestextras",
+          "windows-py39-pluggy",
```

Comment:
Some problems:

* `windows-py39` is used in the `include` section but is missing in `names`
* `ubuntu-py39` is declared in `names` but not used in `include`

From my understanding the we should use `name` here, as this is the variable we want to match in `include` -- perhaps the problems above were the cause for the previous error.

Reply:
Thank you for looking into it, nice catch !